src/components/comment-input.component.js

```diff
@@ -90,6 +92,13 @@ export class CommentInput extends Component {
 
     return (
       <View style={styles.container}>
+        <MentionArea
+          trigger="@"
+          text={this.state.text}
+          updateText={text => this.setState({ text })}
```

Comment:
Hey @andrewda! Can you move this arrow function to a class method instead, if we do it that way we will use the function reference instead of creating a function every time we update the text.